File: voluntarioso-back/Domain/Entities/ONG.cs
namespace Domain.Entities
{
    public sealed class ONG
    {
        public int Id { get; set; }
        public string Email { get; set; } = string.Empty;
        public string Senha { get; set; } = string.Empty;
        public string Nome { get; set; } = string.Empty;
        public string CNPJ { get; set; } = string.Empty;
        public string Categoria { get; set; } = string.Empty;
        public string Missao { get; set; } = string.Empty;
        public string Acoes { get; set; } = string.Empty;
        public string Causa { get; set; } = string.Empty;
        public List<ONGVoluntario> OngVoluntarios { get; set; } = new();
        public List<Voluntario> Voluntarios { get; set; } = new();
    }
}

File: voluntarioso-front/src/models/ONG.ts
import { ONGVoluntario } from './ONGVoluntario';
import { Voluntario } from './Voluntario';

export interface ONG {
    id: number;
    nome: string;
    email: string;
    cnpj: string;
    categoria: string;
    ongVoluntarios: ONGVoluntario[];
    voluntarios: Voluntario[];
    missao: string;
    acoes: string;
    causa: string;
}
